vmui/logs: show read usage in Cardinality Explorer

Loori-R opened the pull request:
### Describe Your Changes

Two new columns were added to the **Cardinality Explorer** table:

`Requests count`
- Displays how many times a metric was queried since stats collection began
- Highlighted **red** when the value is `0` or missing

`Last request`
- Shows when the metric was last queried
- Highlighted:
  - **Red** if older than **30 days**
  - **Yellow** if between **7 and 30 days**
- Shows exact timestamp on hover in `YYYY-MM-DD HH:mm:ss` format

Related issue: #6145

![image](https://github.com/user-attachments/assets/fcf6f33c-e2d0-45c0-9f20-177cf6fa9dde)

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres to [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

hagen1778 commented on app/vmui/packages/vmui/src/constants/dayjsPlugins.ts:
```diff
@@ -1,8 +1,10 @@
 import dayjs from "dayjs";
 import timezone from "dayjs/plugin/timezone";
 import duration from "dayjs/plugin/duration";
```
Would you mind adding the following thing:
> * add a new panel next to Total series named Metric names tracker
> * Metric names tracker should show the number of names in stats tracker statsCollectedRecordsTotal and contain info button.
> * info button should on-hover show a popup with info: statsCollectedSince, cache utilization trackerCurrentMemoryUsageBytes / trackerMemoryMaxSizeBytes * 100 in %, link to https://docs.victoriametrics.com/#track-ingested-metrics-usage

from the https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6145#issuecomment-2710107736 ?

Its purpose to show the stats of the metric name tracker and explain to user what is this feature, where to find docs for it and how to enable it.

Without it it is unclear why user will have 0s everywhere.

Loori-R replied:
Of course, I overlooked `api/v1/status/metric_names_stats` — I’ll add it shortly.

Loori-R commented:
@hagen1778 Updated:

1. Added the **Metric names tracker** panel next to the **Total series**.
2. Updated the table to show `n/a` for the **Requests count** and **Last request** columns when no data is available (previously shown as `0` and `never`).

![image](https://github.com/user-attachments/assets/b12fd4ac-fca4-4e7e-88bb-6b46bc5c8359)
![image](https://github.com/user-attachments/assets/0968bbd5-6b11-44e2-b6b6-67ce4ec0875b)
![image](https://github.com/user-attachments/assets/af795a30-0016-452a-ae80-0393d6ceceda)

hagen1778 reviewed (approved):
Thanks!
